File: ALB/Grundlagen der Informatik 2/Aufgaben/OOAnachCS/OOAnachCS/OOAnachCS/Mitglied.cs
using System;
namespace OOAnachCS
{
    public class Mitglied : Person
    {
        private int nummer;
        private static int anzahlMitglieder = 0;
        private Bankverbindung verbindung;
        
        public Mitglied(string name, string vorname) : base(name,vorname)
        {
            anzahlMitglieder++;
            nummer = anzahlMitglieder;
        }
        public int Nummer
        {
            get { return nummer; }
            set { this.nummer = value; }
        }
        public static int anzeigeAnzahlMitglieder()
        {
            return anzahlMitglieder;
        }
        public Bankverbindung Verbindung
        {
            get { return verbindung; }
            set { this.verbindung = value; }
        }

    }
}

File: ALB/Grundlagen der Informatik 2/C++ Dev-C/OOAnachC++/Mitglied.cpp
//Mitglied.cpp
#include "Mitglied.h"
int Mitglied::anzahlMitglieder = 0;
Mitglied::Mitglied()
{
	anzahlMitglieder++;
}
int Mitglied::anzeigeAnzahlMitglieder()
{
	return anzahlMitglieder;
}
Mitglied::~Mitglied()
{
	delete bank;
}
